Update launcher-related.mdx (#132)
sklauncher/docs: docs/getting-started/installation.mdx

sudo apt update
sudo apt install openjdk-21-jdk
```

> Note: If you get the error ```E: Unable to locate package openjdk-21-jdk``` you may need to configure Debian's APT repository and then run ```sudo apt update```. 

Reviewer: ```suggestion
:::note

If you are using Debian and got the error ```E: Unable to locate package openjdk-21-jdk```, you may need to [upgrade to Debian Testing](https://wiki.debian.org/DebianTesting#How_to_upgrade_to_Debian_.28next-stable.29_Testing) or use an older Java version, such as ```openjdk-17-jdk```.

:::
```
Your instructions are unclear. We should give more information on how to solve the problem, and also let people know that they are for Debian GNU/Linux, because [mixing Ubuntu and Debian packages is not a wise thing to do and could damage the user's system](https://askubuntu.com/questions/264764/how-can-i-install-packages-from-debians-squeeze-repo-on-ubuntu-12-10/264766#264766). This is in case user gets this error in Ubuntu and decides that adding a Debian repo to `sources.list` could remedy the problem.

Author: > Your instructions are unclear. We should give more information on how to solve the problem, and also let people know that they are for Debian GNU/Linux, because [mixing Ubuntu and Debian packages is not a wise thing to do and could damage the user's system](https://askubuntu.com/questions/264764/how-can-i-install-packages-from-debians-squeeze-repo-on-ubuntu-12-10/264766#264766). This is in case user gets this error in Ubuntu and decides that adding a Debian repo to `sources.list` could remedy the problem.

This makes much more sense. I changed it. Anyways, my bad, I should've made the instructions clear 😅

Thank you :D
